Android/Phone_Client/app/src/main/java/com/example/asymetricasteroids/Client.kt

```diff
@@ -1,66 +1,52 @@
 package com.example.asymetricasteroids
 
-import android.os.Handler
-import android.os.Looper
-import android.util.Log
-import java.io.IOException
-import java.io.InputStream
-import java.io.OutputStream
-import java.net.InetAddress
-import java.net.InetSocketAddress
+import net.razorvine.pickle.Pickler
+import net.razorvine.pickle.Unpickler
 import java.net.Socket
-import java.util.concurrent.Executors
 
-class Client(hostAddress: InetAddress): Thread() {
-
-    var hostAddress: String = hostAddress.hostAddress
-    lateinit var inputStream: InputStream
-    lateinit var outputStream: OutputStream
-    lateinit var socket: Socket
-
-    fun write(byteArray: ByteArray){
-        try {
-            outputStream.write(byteArray)
-        }catch (ex: IOException){
-            ex.printStackTrace()
+class Client(addr : String, port: Int) {
+    // Needs to be changed depending on the threadedServer.py
+    val SERVER_ADDRESS = addr
```

Comment:
Preferably would make this thing a StringResource, but whatever, just recompile every time you have to change it lol